packages/@sanity/form-builder/src/inputs/arrays/ArrayOfObjectsInput/InsertMenu.tsx

```diff
@@ -9,7 +9,12 @@ interface Props {
   onInsert: (pos: 'before' | 'after', type: SchemaType) => void
 }
 
-const MENU_POPOVER_PROPS = {portal: true, tone: 'default', placement: 'left'} as const
+const MENU_POPOVER_PROPS = {
```

Comment:
Nice! It would be nice to get typings for this item:

```ts
const MENU_POPOVER_PROPS: MenuGroupProps['popover'] = {
  constrainSize: true,
  placement: 'left',
  portal: true,
  tone: 'default',
}
```

Reply:
Good point! I've added it :)